integrationtests/src/test/java/org/axonframework/integrationtests/commandhandling/EventPublicationOrderTest.java

```diff
@@ -63,7 +63,7 @@ void setUp() {
     void testPublicationOrderIsMaintained_AggregateAdded() {
         String aggregateId = UUID.randomUUID().toString();
         GenericDomainEventMessage<StubAggregateCreatedEvent> event =
-                new GenericDomainEventMessage<>("test", aggregateId, 0, new StubAggregateCreatedEvent(aggregateId));
```

Comment:
Does this test work without these changes? If so, I suggest not doing this change. If not, we probably have a backward compatibility issue.